driver-bundle/src/main/java/com/microsoft/playwright/impl/driver/jar/DriverJar.java

```diff
@@ -140,7 +151,7 @@ void extractDriverToTempDir() throws URISyntaxException, IOException {
           }
           toPath.toFile().deleteOnExit();
         } catch (IOException e) {
-          throw new RuntimeException("Failed to extract driver from " + uri + ", full uri: " + originalUri, e);
```

Comment:
nit: having originalUri in the exception was useful for debugging when the jar was backed into some other jar (see https://github.com/microsoft/playwright-java/issues/306#issuecomment-797355207). We can live without it but maybe we can preserve it by using something like getDriverResourceURI in the test instead of getPlatformDriver and calling maybeExtractNestedJar only inside extractDriverToTempDir (this is for an edge case scenarios anyway and shouldn't affect your test).